buildenv/jenkins/tapVerification/tapVerification.groovy

```diff
@@ -0,0 +1,75 @@
+#!groovy
+
+def AQA_REPO = params.AQA_REPO ?: "adoptium"
+def AQA_BRANCH = params.AQA_BRANCH ?: "master"
```

Comment:
should that be main these days?

Reply:
yes, it is master. https://github.com/adoptium/aqa-tests/tree/master